e2e/volumes/input/volumes.nomad

```diff
@@ -0,0 +1,104 @@
+job "volumes" {
+  datacenters = ["dc1", "dc2"]
+
+  group "group" {
+
+    volume "data" {
+      type   = "host"
+      source = "shared_data"
+    }
+
+    task "docker_task" {
+
+      driver = "docker"
+
+      config {
+        image   = "busybox:1"
+        command = "/bin/sh"
+        args    = ["/usr/local/bin/myapplication.sh"]
+
+        mounts = [
+          # this mount binds the task's own NOMAD_TASK_DIR directory as the
+          # source, letting us map it to a more convenient location; this is a
+          # frequently-used way to get templates into an arbitrary location in
+          # the task for Docker
```

Comment:
h/t to @angrycub for this one; this job should serve as a good starting point to expand the documentation for #8919